src/Features/CSharp/Portable/QuickInfo/CSharpSemanticQuickInfoProvider.cs

```diff
@@ -151,6 +152,10 @@ protected override NullableFlowState GetNullabilityAnalysis(SemanticModel semant
         {
             return null;
         }
+        if (document.IsRazorDocument())
```

Comment:
this really needed a comment explaning wy we would nto get tehse results in razor.